internal/driver/gomobile/menu_test.go

```diff
@@ -5,12 +5,48 @@ package gomobile
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	"fyne.io/fyne"
```

Comment:
I thought the testify imports should go behind the fyne.io imports. 😕 
However, I really don’t care as long as `gofmt` or `goimports` don’t touch it 😃.

Reply:
Oh yes you are right. I think I reset my goland config by mistake.

Reply:
Fixed :)